catalogs/catalog-common/src/main/java/org/apache/gravitino/catalog/lakehouse/iceberg/IcebergConstants.java

```diff
@@ -84,4 +84,6 @@ public class IcebergConstants {
 
   public static final String ICEBERG_REST_DEFAULT_METALAKE = "gravitino";
   public static final String ICEBERG_REST_DEFAULT_CATALOG = "default_catalog";
+
+  public static final String ICEBERG_VERSION = "1.6.1";
```

Comment:
This hardcoded version number seems not flexible enough. I found that it is only used in ITs, can we just:

1. use an environment variable as the version number, and pass in this environment variable from the gradle task.
2. The iceberg version number can be figured out in gradle code using `version.toml` definition.

With the above changes, we only need to modify the version number in `libs.versions.toml`.

Reply:
done